buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -237,7 +237,7 @@ def test(JOB_NAME, UPSTREAM_JOB_NAME, UPSTREAM_JOB_NUMBER, NODE, OPENJ9_REPO, OP
             string(name: 'VENDOR_TEST_DIRS', value: VENDOR_TEST_DIRS),
             string(name: 'USER_CREDENTIALS_ID', value: USER_CREDENTIALS_ID),
             string(name: 'TEST_FLAG', value: TEST_FLAG),
-            string(name: 'KEEP_REPORTDIR', value: keepReportDir),
+            booleanParam(name: 'KEEP_REPORTDIR', value: keepReportDir),
```

Comment:
To be safe, can we have `keepReportDir.toBoolean()`?

Reply:
I suppose that is more simple and safer. Testing it out...

Reply:
Still works